jetstream/options.go

```diff
@@ -301,23 +317,26 @@ func WithExpectStream(stream string) PublishOpt {
 	}
 }
 
-// WithExpectLastSequence sets the expected sequence in the response from the publish.
+// WithExpectLastSequence sets the expected sequence in the response from the
+// publish.
```

Comment:
I think the description isnt clear - this only publishes the message if the last message in the stream has this seq

Reply:
Changed to:
```go
// WithExpectLastSequence sets the expected sequence number the last message
// on a stream should have. If the last message has a different sequence number
// server will reject the message and publish will fail.
```